vcf2maf.pl

```diff
@@ -233,10 +228,38 @@ sub GetBiotypePriority {
     'retain-ann=s' => \$retain_ann,
     'min-hom-vaf=s' => \$min_hom_vaf,
     'remap-chain=s' => \$remap_chain,
-    'max-subpop-af=f' => \$max_subpop_af
+    'max-subpop-af=f' => \$max_subpop_af,
+    'samtools-exec=s' => \$samtools,
+    'tabix-exec=s' => \$tabix,
+    'liftover-exec=s' => \$liftover,
+
 ) or pod2usage( -verbose => 1, -input => \*DATA, -exitval => 2 );
+
+if( $man ) {
+    pod2usage( -verbose => 2, exitval => 0)
+}
 pod2usage( -verbose => 1, -input => \*DATA, -exitval => 0 ) if( $help );
-pod2usage( -verbose => 2, -input => \*DATA, -exitval => 0 ) if( $man );
+
+# Search the PATH for samtools and tabix, unless specified on the command line.
+# Error out if either is missing from PATH or the specified exec doesn't exist.
+
+( $samtools ) = map{chomp; $_}`which samtools` unless ( $samtools ) ;
+( $samtools ) or die "ERROR: Please install samtools on your PATH, or specify --samtools-exec\n";
+( -e $samtools ) or die "ERROR: Specified samtools: <$samtools> does not exist\n";
+
+( $tabix ) = map{chomp; $_}`which tabix` unless ( $tabix );
+( $tabix ) or die "ERROR: Please install tabix on your PATH, or specify --tabix-exec\n";
+( -e $tabix ) or die "ERROR: Specified tabix: <$tabix> does not exist\n";
+
+if( $remap_chain ) {
```

Comment:
The detection of `$remap_chain` and confirmation that `$liftover` is set is done up front so we don't have to wait to see if the `liftOver` executable exists.